.changeset/long-sloths-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'myst-cli': patch
```

Comment:
A few other changesets need to be captured! frontmatter, transforms and tex-to-myst. I will add them now.